cli.js

```diff
@@ -20,7 +20,11 @@ const executeScript = (script, args) => {
   const ext = path.extname(script)
   const program = extProgramMap[ext]
   if (!program) throw new Error(`ta-script has no program assigned to handle the "${ext}" ext`)
-  cp.execSync(`${program} ${script} ${args}`, {stdio: 'inherit'})
+  try {
+    cp.execSync(`${program} ${script} ${args}`, {stdio: 'inherit'})
+  } catch (e) {
+    process.exit(1)
```

Comment:
This is still exiting with 0:

![image](https://cloud.githubusercontent.com/assets/5067638/12761671/bd307398-c9a0-11e5-89af-0569688b4c41.png)